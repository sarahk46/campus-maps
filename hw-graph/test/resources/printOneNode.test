# Tests that a graph can be created, a node can be added to it,
# and it is possible to list the added node.

# Create a graph
CreateGraph graph1

# Add one node
AddNode graph1 n1

# Print the node in the graph
ListNodes graph1