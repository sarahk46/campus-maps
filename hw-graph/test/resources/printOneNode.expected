# Tests that a graph can be created, a node can be added to it,
# and it is possible to list the added node.

# Create a graph
created graph graph1

# Add one node
added node n1 to graph1

# Print the node in the graph
graph1 contains: n1